esp/services/ws_fs/ws_fsService.cpp

```diff
@@ -3661,15 +3662,15 @@ bool CFileSprayEx::onGetDFUServerQueues(IEspContext &context, IEspGetDFUServerQu
 {
     try
     {
-#ifdef _CONTAINERIZED
-        UNIMPLEMENTED_X("CONTAINERIZED(CFileSprayEx::onGetDFUServerQueues)");
-#else
         context.ensureFeatureAccess(FILE_SPRAY_URL, SecAccess_Read, ECLWATCH_FILE_SPRAY_ACCESS_DENIED, "Permission denied.");
 
         StringArray qlist;
+#ifdef _CONTAINERIZED
+        qlist.append(m_QueueLabel); // TODO: re-design support for multiple queues (see the comment in init()).
```

Comment:
This may as well be implemented using code similar to init().  Something like:

```
    Owned<IPropertyTreeIterator> dfuQueues = getComponentConfigSP()->getElements("dfuQueues");
    ForEach(*dfuQueues)
    {
        IPropertyTree & dfuQueue = dfuQueues->query();
        const char * dfuName = dfuQueue.queryProp("@name");
        assertex(!isEmptyString(dfuName)); //  redundant since checked by the schema...
        StringBuffer queueLabel;
        getDfuQueueName(queueLabel, dfuName);
        qlist.append(queueLabel);
    }

```

Reply:
OK, I will change.